state/machine_internal_test.go

```diff
@@ -0,0 +1,57 @@
+package state
+
+import (
+	gc "gopkg.in/check.v1"
+	"gopkg.in/mgo.v2/bson"
+	"gopkg.in/mgo.v2/txn"
+)
+
+type MachineInternalSuite struct {
+}
+
+var _ = gc.Suite(&MachineInternalSuite{})
+
+func (s *MachineInternalSuite) TestCreateUpgradePrepareLockTxnAssertsMachineAlive(c *gc.C) {
+	arbitraryId := "1"
+	arbitraryData := &upgradeSeriesLock{}
+	for _, op := range createUpgradeSeriesPrepareLockTxnOps(arbitraryId, arbitraryData) {
```

Comment:
Reading through all this, it should probably all just be switched to either
prepareUpgradeSeriesLock
or
createUpgradeSeriesLock

but not UpgradeSeriesPrepareLock.